### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

File: sonar-runner-api/src/main/java/org/sonar/runner/api/ScanProperties.java [hunk 1]
```diff
@@ -31,7 +31,9 @@
    * Default task
    *
    * @see RunnerProperties#TASK
+   * @deprecated since 2.5 No more task since SQ 5.2
    */
+  @Deprecated
   String SCAN_TASK = "scan";
```
Comment (SonarTech) [inline sonar-runner-api/src/main/java/org/sonar/runner/api/ScanProperties.java]: INFO Do not forget to remove this deprecated code someday. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)

File: sonar-runner-api/src/main/java/org/sonar/runner/api/RunnerProperties.java [hunk 2]
```diff
@@ -33,7 +33,9 @@
 
   /**
    * Task to execute, "scan" by default
+   * @deprecated since 2.5 No more task starting from SQ 5.2
    */
+  @Deprecated
   String TASK = "sonar.task";
```
Comment (SonarTech) [inline sonar-runner-api/src/main/java/org/sonar/runner/api/RunnerProperties.java]: INFO Do not forget to remove this deprecated code someday. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues:
* INFO 2 info

Watch the comments in this conversation to review them.

**Comment by dbmeneses** [comment]
LGTM

**Pull request merged**